src/ontology/cl-edit.owl

```diff
@@ -22464,6 +22466,24 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0019019 "smooth muscle cell
 AnnotationAssertion(rdfs:label obo:CL_0019019 "tracheobronchial smooth muscle cell"@en)
 EquivalentClasses(obo:CL_0019019 ObjectIntersectionOf(obo:CL_0000192 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0007196)))
 
+# Class: obo:CL_0019031 (intestine goblet cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:1996606") Annotation(oboInOwl:hasDbXref "PMID:24942678") obo:IAO_0000115 obo:CL_0019031 "Goblet cells reside throughout the length of the small and large intestine and are responsible for the production and maintenance of the protective mucus blanket by synthesizing and secreting high-molecular-weight glycoproteins known as mucins. The intestinal goblet cells secrete the MUC2 mucin, as well as a number of typical mucus components: CLCA1, FCGBP, AGR2, ZG16, and TFF3.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_0019031 <http://orcid.org/0000-0003-2034-601X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_0019031 "2020-10-14T14:59:07Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:1996606") oboInOwl:hasExactSynonym obo:CL_0019031 "intestinal goblet cell")
+AnnotationAssertion(rdfs:label obo:CL_0019031 "intestine goblet cell"@en)
+EquivalentClasses(obo:CL_0019031 ObjectIntersectionOf(obo:CL_0000160 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000160)))
+
+# Class: obo:CL_0019032 (intestinal tuft cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22527717") obo:IAO_0000115 obo:CL_0019032 "Any brush cell that is part of the intestine.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> obo:CL_0019032 <http://orcid.org/0000-0003-2034-601X>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> obo:CL_0019032 "2020-10-15T12:47:46Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22527717") oboInOwl:hasExactSynonym obo:CL_0019032 "intestinal brush cell")
+AnnotationAssertion(rdfs:label obo:CL_0019032 "intestinal tuft cell"@en)
+EquivalentClasses(obo:CL_0019032 ObjectIntersectionOf(obo:CL_0002204 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0000160)))
```

Comment:
Make uberon term more specific - http://purl.obolibrary.org/obo/UBERON_0001277 'intestinal epithelium' looks best to me.

Reply:
Thanks, done in 18cecee0f0c2b421ddadd33034c271ab1a551bb8 👍